tools/aquainfra_ogc_api_processes/macros.xml

```diff
@@ -41,7 +41,7 @@
                     <param name="input_data" value="points_att_polygon_test_input_3.txt"/>
                     <param name="colname_long" value="longitude"/>
                     <param name="colname_lat" value="latitude"/>
-                    <param name="colname_value_name" value="transparency_m"/>
+                    <param name="colname_value_name" value="transparen"/>
```

Comment:
mh, this looks wired, can you double check?

Reply:
That's indeed correct since the column in the corresponding data is called "transparen" (for whatever reason :) )